sematic/ui/packages/common/src/pages/RunSearch/RunListEmptyState.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "@emotion/styled";
+import Typography from "@mui/material/Typography";
+import MuiRouterLink from "src/component/MuiRouterLink";
+import { ShellCommandRelaxed } from "src/component/ShellCommand";
+import theme from "src/theme/new";
+
+const InitialGuide = styled.div`
+    width: 806px;
+    height: 317px;
+    padding: ${theme.spacing(10)};
+
+    border: 1px solid ${theme.palette.p3border.main};
+    border-radius: 5px;
+`
+
+const StyledTypography = styled(Typography)`
+    margin-top: ${theme.spacing(5)};
+    margin-bottom: ${theme.spacing(5)};
+`
+
+const StyledLink = styled(MuiRouterLink)`
+    color: ${theme.palette.primary.main};
+    font-size: ${theme.typography.big.fontSize}px;
+`;
+
+const StyledTitle = styled.h1`
+    font-size: 24px;
+    font-weight: 500;
+    & a {
+        font-size: 24px;
+        font-weight: 500;
+    }
+`
+
+
+export const NoRunNoFilters = () => <InitialGuide >
```

Comment:
I believe this get started page should only be displayed when there are no runs at all in the DB, in addition to no filters being set.

Reply:
That's correct.